src/components/Navbar/components/UserSelector/index.tsx

```diff
@@ -18,9 +18,8 @@ export default function UserSelector() {
 
   const { user, refresh, refreshLibrary } = React.useContext(ContextProvider)
   const handleLogout = async () => {
-    if (confirm('are you sure?')) {
-      await legendary(`auth --delete`)
-      await legendary(`cleanup`)
+    if (confirm('Are you sure?')) {
```

Comment:
Hmm, one label we missed translating.
can you add the label here with this text as default and then we can remember translating it?

Reply:
Sure.